x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/settings/install_button.tsx

```diff
@@ -46,13 +48,33 @@ export function InstallButton(props: InstallationButtonProps) {
     setIsInstallModalVisible(!isInstallModalVisible);
   }, [isInstallModalVisible]);
 
+  const { notifications } = useStartServices();
+
   const handleClickInstall = useCallback(async () => {
     setFleetSetupInProgress(true);
     toggleInstallModal();
-    await sendPostFleetSetup({ forceRecreate: false });
+    const res = await sendPostFleetSetup({ forceRecreate: false });
```

Comment:
a little nitpick but I would put this in a try catch to avoid uncaught exception here.
```js
try {
const res = await sendPostFleetSetup({ forceRecreate: false });
if (res.error) {
 throw res.error;
}
} catch (e) {
 // show toast
}

```